nikola/plugins/compile/ipynb/__init__.py

```diff
@@ -32,7 +32,15 @@
 
 try:
     from IPython.nbconvert.exporters import HTMLExporter
-    from IPython.nbformat import current as nbformat
+    import IPython.nbformat
+    if IPython.version_info[0] >= 3:     # API changed with 3.0.0
+        def nbformat_reads(s):
+            # Update to the current format so that the export works.
+            return IPython.nbformat.reads(
+                s, IPython.nbformat.current_nbformat)
+    else:
+        nbformat_reads = IPython.nbformat.current.reads_json
```

Comment:
Can’t you just use plain `reads` on all versions?

```py
nb_json = IPython.nbformat.reads(s)
```

seems to be enough on v2 and v3, at least according to the docs.